iocore/eventsystem/I_EThread.h

```diff
@@ -56,7 +56,7 @@ enum ThreadType {
   DEDICATED,
 };
 
-extern volatile bool shutdown_event_system;
+extern bool shutdown_event_system;
```

Comment:
`volatile` shouldn't have any effect.  It is used incorrectly throughout ATS as a synchronization mechanism.